### PR TITLE
New IBAN Countries and upgrade version ISO 13616

### PR DESCRIPTION
**Pull request by pmtitou**
Update / add bban format for countries below :
CR - BY - LC - SC - ST - SV

### CODE REVIEW

**Comment by arturmkrtchyan** [comment]
@pmtitou thanks for the fixes and sorry for the delay. Could you please add a reference of doc that supports the structure changes and the new countries ?

I will double check merge and release a new version

**Comment by behnaaz** [comment]
Any idea when this get merged?

File: src/main/java/org/iban4j/bban/BbanStructure.java [hunk 1]
```diff
@@ -348,13 +359,26 @@ private BbanStructure(final BbanStructureEntry... entries) {
                 new BbanStructure(
                         BbanStructureEntry.bankCode(4, 'a'),
                         BbanStructureEntry.accountNumber(21, 'c')));
+        
+        structures.put(CountryCode.SC,
+                new BbanStructure(
+                		BbanStructureEntry.bankCode(4, 'a'),
+    					BbanStructureEntry.branchCode(4, 'n'),
+                        BbanStructureEntry.accountNumber(16, 'n'),
+                        BbanStructureEntry.accountType(3, 'a')));
```
Comment (bvaardal) [inline src/main/java/org/iban4j/bban/BbanStructure.java:368]: The problem is that the bankCode is actually `4!a2!n` and the branchCode is `2!n`, but we can't support the mixed representation.

Is it better to say bankCode is 6!c and the branchCode is 2!n?

Comment (bvaardal) [reply src/main/java/org/iban4j/bban/BbanStructure.java:368]: bank.code appears to agree with this code.

**Comment by bvaardal** [comment]
Sources:
https://www.swift.com/resource/iban-registry-pdf
https://bank.codes/iban/structure

**Pull request merged**